app/models/search-result.ts

```diff
@@ -245,7 +245,7 @@ export default class SearchResultModel extends Model {
     get orcids() {
         if (this.resourceMetadata.identifier) {
             const orcids = this.resourceMetadata.identifier.filter(
-                (item: any) => item['@value'].includes('http://orcid.org/'),
+                (item: any) => item['@value'].includes('orcid.org/'),
```

Comment:
```suggestion
                (item: any) => new URL(item['@value']).host === 'orcid.org',
```
(the more we use this iri-focused api, the more i think we need a small/good set of iri/rdf utils to simplify -- tho for now builtins help)